### ISSUE TITLE
Perceivable Chapter 1 Exam in Section 508 course

### ISSUE THREAD

**Issue by AnnMarieDavis** [Section508Coordinators/DHSA-TT-200-A#409]
## Issue Type
- [ ] System Bug (the course doesn't function properly)
- [ ] Typo/grammar
- [X] Incorrect content
- [ ] 508 issue (something is not accessible according to our own test process)
- [ ] Other, specify...[replace this text with your issue type description]

## Description
In the quiz for “Perceivable Chapter 1 Exam”, why is a AAA SCs included? 
Also the answer seemed to be incorrect.  I think it should have been “a” based on WCAG “This is done by periodically freezing the synchronized media presentation and playing additional audio description. The synchronized media presentation is then resumed."

## Location
https://www.w3.org/WAI/WCAG21/Understanding/extended-audio-description-prerecorded.html  

## Actual Behavior / Observation
![image](https://user-images.githubusercontent.com/31414225/79337440-84e56400-7ef3-11ea-8e3e-77e82913cce8.png)

## Expected Behavior / Recommendation
Consider revising the question
Issue submitted by Shou-Ching Schilling <SHOUCL_MS@msn.com>

**Comment by AnnMarieDavis** [comment]
Kristen Smith's comment - Disagree and reject. We don't follow AAA per Section 508, and the answer of A also just can't be right as we wrote it. We mention nothing of ADs in the answer, just that people can pause and restart. B is the obvious best choice.
